cyvlfeat/sift/test_phow.py

```diff
@@ -0,0 +1,5 @@
+from unittest import TestCase
```

Comment:
We are using nose for testing and so you need need `TestCase`. Just make sure the test has the word `test` in the name.